libraries/SPI/src/SPI.h

```diff
@@ -304,4 +310,18 @@ class SPIClass {
 
 extern SPIClass SPI;
 
+#if defined(SUBGHZSPI_BASE)
+class SUBGHZSPIClass : public SPIClass {
+  public:
+    SUBGHZSPIClass(): SPIClass{SUBGHZSPI_MOSI, SUBGHZSPI_MISO, SUBGHZSPI_SCLK, SUBGHZSPI_SS} {}
+
+    void begin(uint8_t _pin = CS_PIN_CONTROLLED_BY_USER);
+    byte transfer(uint8_t _pin, uint8_t _data, SPITransferMode _mode = SPI_LAST);
```

Comment:
These functions are not virtual, so if a sketch passes the `SUBGHZSPIClass` object through `SPIClass` pointer or reference (which is a common pattern with libraries), the original methods will be called instead of these. I guess we need to make them virtual in `SPIClass` to make this work in all cases?

Reply:
done